test/TimelockMultiAdminShim.unit.t.sol

```diff
@@ -481,8 +481,7 @@ contract ExecuteTransaction is TimelockMultiAdminShimTest {
     vm.prank(admin);
 
     timelockMultiAdminShim.executeTransaction{value: _value}(_target, _value, _signature, _data, _eta);
-
-    assertEq(address(timelockMultiAdminShim).balance, _value);
+    assertEq(address(timelockMultiAdminShim).balance, 0);
```

Comment:
This test does not fully exercise the actual functionality that we changed here, namely that extra eth that was in the shim beforehand (presumably because someone accidentally sent it) would be flushed. We should pull out an issue to add a test for this separately. This PR Is good to merge though, as it does address and fix the issue.

Reply:
Added a new test in this PR